packages/css/src/grid/grid.scss

```diff
@@ -20,3 +22,44 @@
     grid-template-columns: repeat(var(--amsterdam-grid-wide-column-count), 1fr);
   }
 }
+
+.amsterdam-grid--density-high {
+  --amsterdam-grid-gap: var(--amsterdam-grid-density-high-gap);
+  --amsterdam-grid-padding-inline: var(--amsterdam-grid-density-high-padding-inline);
+}
+
+.amsterdam-grid--margin-bottom--small {
+  padding-block-end: calc(var(--amsterdam-grid-gap) / 2);
```

Comment:
I believe @alimpens already said this but the class is "margin" and the rule is "padding"?

Reply:
Changed it to padding now.